packages/components/src/theme/charts.scss

```diff
@@ -112,7 +112,50 @@
         overflow: hidden;
     }
 }
-
-.r-report__images img {
+.chart-body {
+    position: relative;
+}
+.chart-panel {
+    margin-bottom: 8px;
+    min-height: 100px;
```

Comment:
Design feedback. On a large screen the SVG images seem to get too large. Should we consider a maximum height for these?

<img width="2494" alt="image" src="https://github.com/LabKey/labkey-ui-components/assets/3926239/592c0f5c-f5d8-42c6-bb1d-db5c1aa5832d">

Reply:
I have a fix for this locally.